### PR TITLE
Fix template README.md admonition

### PR DESCRIPTION
**Pull request by scwatts**
## Changes

- multi-line GH style admonitions are not supported by octokit.js ([link](https://github.com/nf-core/website/blob/cf6fb88/src/components/octokit.js#L83-L99))
  - see https://github.com/nf-core/oncoanalyser/pull/41 for a little more context
- the above constraint causes malformed rendering of a template README.md admonition on the website (**Figure 1**)
- this PR resolves by converting the multi-line admonition into a single-line admonition

**Figure 1.** Malformed multi-line GH style admonition from the nf-core website

image

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Comment by scwatts** [comment]
Force pushing after rebasing on top of correct target branch.

**Comment by scwatts** [comment]
Seeing the following error from `nf-core lint`:

```
╭─ [✗] 1 Pipeline Test Failed ─────────────────────────────────────────────────╮
│                                                                              │
│ actions_schema_validation: Failed to fetch schema: Response code for         │
│ https://json.schemastore.org/github-workflow was 503                         │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
```

**Comment by mashehu** [comment]
Yes, schemastore is unfortunately not the most stable website at the moment. See discussion on slack

**Review by mashehu — APPROVED** [review]
Thanks for this catch

File: nf_core/pipeline-template/README.md [hunk 1]
```diff
@@ -80,8 +80,7 @@ nextflow run {{ name }} \
 ```
 
 > [!WARNING]
-> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_;
-> see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
+> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_; see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
```
Comment (ewels) [inline nf_core/pipeline-template/README.md:83]: Updating the link after the recent docs rearrangement in the website:
```suggestion
> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_; see [docs](https://nf-co.re/docs/usage/getting_started/configuration#custom-configuration-files).
```

**Comment by scwatts** [comment]
Previously seen error in the `Run test_components.py with Python 3.12 on ubuntu-latest` check, I think this is the offending action:

```
│   Command error:                                                             │
│     Unable to find image 'quay.io/biocontainers/bwa:0.7.17--hed695b0_7'      │
│ locally                                                                      │
│     0.7.17--hed695b0_7: Pulling from biocontainers/bwa                       │
│     docker: [DEPRECATION NOTICE] Docker Image Format v1 and Docker Image     │
│ manifest version 2, schema 1 support is disabled by default and will be      │
│ removed in an upcoming release. Suggest the author of                        │
│ quay.io/biocontainers/bwa:0.7.17--hed695b0_7 to upgrade the image to the OCI │
│ Format or Docker Image manifest v2, schema 2. More information at            │
│ [https://docs.docker.com/go/deprecated-image-specs/.](https://docs.docker.com/go/deprecated-image-specs/)                          │
│     See 'docker run --help'.                                                 │
```

**Comment by mirpedrol** [comment]
The error was fixed now, it can be merged 🙂

**Pull request merged**